doc/src/user_guide.rst

```diff
@@ -39,6 +39,16 @@ or greater. The backend requires:
 1. `CUDA <https://developer.nvidia.com/cuda-downloads>`_ >= 4.2
 2. `pycuda <http://mathema.tician.de/software/pycuda/>`_ >= 2011.2
 
+MIC Backend
+^^^^^^^^^^^
+
+The MIC backend targets Intel Xeon Phi co-processors. The backend
+requires:
+
+1. ICC >= 16.0
```

Comment:
Do we need a version this recent?  I thought that Jin Seok was having trouble with recent releases?